commands/access/remove.js

```diff
@@ -1,8 +1,28 @@
 'use strict';
 
-var Heroku      = require('heroku-client');
-var co          = require('co');
-var heroku;
+let cli           = require('heroku-cli-util');
+let Utils         = require('../../lib/utils');
+let co            = require('co');
+
+function* run(context, heroku) {
+  let appName = context.app;
+  let appInfo = yield heroku.apps(appName).info();
+  let request;
+
+  if (Utils.isOrgApp(appInfo.owner.email)) {
+    request = heroku.request({
+      method: 'DELETE',
+      path: `/organizations/apps/${appName}/collaborators/${context.args.email}`,
```

Comment:
AFAIK, deleting a collaborator is the same. The difference between the org collab and regular collab endpoints are updating user privileges.